template.yml.erb

```diff
@@ -401,6 +412,8 @@ Resources:
             ZUbJMN3hMFIFP4smAtjkKqguLJPFdcMij7WFDjL5kPQpxuc4RSq97iXRODIBjW1B\n zdN9ZpGRA4MDOu3BtLC69PwoFrSULAzlUmC5fJIwpaXoRGZEYRMQThlJ9VsRGwTG\n
             JTJJ4dWJ2cw61prbeU/HVCWD8OaNumz9tQIDAQAB\n
             -----END RSA PUBLIC KEY-----'
+          limit_per_hour: !Ref LimitPerHour
+          limit_per_day: !Ref LimitPerDay
```

Comment:
NIT: Intentional to add these to both lambdas? (Not sure that there's a straightforward way to omit this from just the WebSocket Lambda, so there may not be anything to do, but it may be worth adding a comment in case these get split out in the future)

Reply:
Yeah there might? be a way to add just to http, but think it's probably more confusing than it'd be worth. I'll add a comment.